### DISCUSSION TITLE
Formatting DLRS Date Results

### DISCUSSION THREAD

**Discussion by jnoden** [SFDO-Community/declarative-lookup-rollup-summaries#1144] [Q&A]
I'm trying to concatenate the dates of all payments on an opportunity.  The payment date in the child object is formatted correctly (mm/dd/yyyy), but when I run the DLRS the resulting string formats the date(s) as date/time (yyyy-mm-dd 00:00:00).  Is there a way to format the result to match the date format on the payment records?

Current result looks like:  2021-03-15 00:00:00;2022-01-12 00:00:00
Successful result should look like:  3/15/2021;1/12/2022

**Comment by aheber** [comment]
DLRS doesn't offer a way to format the results at this time.
